src/rules/no-empty-definitions.js

```diff
@@ -8,7 +8,7 @@
 //-----------------------------------------------------------------------------
 
 /**
- * @typedef {import("../types.ts").MarkdownRuleDefinition<{ RuleOptions: []; }>}
+ * @typedef {import("../types.ts").MarkdownRuleDefinition<{ RuleOptions: [{ checkFootnoteDefinitions?: boolean; }] }>}
```

Comment:
```suggestion
 * @typedef {import("../types.ts").MarkdownRuleDefinition<{ RuleOptions: [{ checkFootnoteDefinitions?: boolean; }] }>}
```

Could you merge the `main` branch and update the code to use the `@import` JSDoc syntax as shown in the ongoing changes in https://github.com/eslint/markdown/pull/367/files?

You can refer to the following image as a guide:

![image](https://github.com/user-attachments/assets/1313f60d-07fa-422a-92f9-9c5c2b506fe1)